src/index.js

```diff
@@ -1,39 +1,108 @@
 import './style.css';
 
 const todoList = document.getElementById('todoList');
+const addListForm = document.querySelector('#form1');
+const textInput = document.getElementById('textInput');
 
-const taskData = [
-  {
-    description: 'wash the dishes',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: '  Complete To Do list project',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'Attend family meeting',
-    completed: true,
-    index: 3,
-  },
-];
+let todos = JSON.parse(localStorage.getItem('todos'));
+// display task
+const displayTask = () => {
+  let li = '';
+  if (todos) {
+    todos.forEach((todo) => {
+      const completed = todo.completed ? 'checked' : '';
+      li += `
+      <div class="checkContainer">
+      <div class="inputContainer">
+          <input id="checkBox" type="checkbox"  ${completed} data-check="${todo.index}">
+          <input id="label" data-desc="${todo.index}" value="${todo.description}">
+      </div>
+      <div class="btnContainer">
+          <button><i id="${todo.index}" class="fa fa-ellipsis-v btnDelete"></i></button>    
+      </div>
+    </div>`;
+    });
+  }
+  todoList.innerHTML = li;
+};
+displayTask();
 
-let list = '';
-taskData.forEach((data) => {
-  list += `
+// add task
+const addList = (e) => {
+  e.preventDefault();
+  const userInput = textInput.value.trim();
+  textInput.value = '';
+  if (!userInput) return;
+  if (!todos) {
+    todos = [];
+  }
+  const list = {
+    description: userInput,
+    completed: false,
+    index: todos.length + 1,
+  };
+  todos.push(list);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  displayTask();
+};
+addListForm.addEventListener('submit', addList);
 
+// remove task
+const removeList = (index) => {
+  const newlist = todos.filter((element) => element.index !== index);
+  todos.length = 0;
+  let i = 1;
```

Comment:
- [ ] Good job 👏 👏 Your Add and Remove features are working. However, from the project requirements, they should be in a separate file. In other words, you are supposed to create a module containing add and remove functionality and then import it here in the `index.js` file for use 😃 👍  This is what looks more professional🏃 
This is the highlight among the project requirements 👇 
![image](https://user-images.githubusercontent.com/90222110/194714440-488730c7-5853-4cfb-8244-e9542638e3f5.png)